git-icdiff

```diff
@@ -10,8 +10,8 @@ if [ -z "$GITPAGER" ]; then
   GITPAGER="less"
 fi
 
-if [ "$GITPAGER" = "more" -o "$GITPAGER" = "less" ]; then
+if [ "$GITPAGER" = "more" ] -o [ "$GITPAGER" = "less" ]; then
```

Comment:
This doesn't make sense.  `-o` only works inside `[...]`.

I guess we could switch it to `if [ ... ] || [ ... ]` but the current version is fine.

Reply:
Oops, amended.